src/components/MyData/LineChart/DataSelector.tsx

```diff
@@ -168,7 +165,7 @@ const CustomLabel = ({ selection, x = 0, y = 0 }: CustomLabelProps) => {
   const { styles } = useStyles(defaultChartStyles);
 
   const decreaseBy = !selection.point1 || !selection.point2 ? 25 : 0;
-  const shiftX = Math.min(selection.chartWidth - (x + 115 - decreaseBy), 0);
+  const shiftX = Math.min(selection.chartWidth - (x + 15 - decreaseBy), 0);
```

Comment:
Nit-pick: What is this `15` and is there any way to reference it?  Just giving it a variable name, and opening up the possibility of future reuse, might be nice.

Reply:
Yeah great question, it is an amount to shift the tooltip by so that it doesn't get cut off. Let me see if I can make this a bit more obvious.